bikes/views.py

```diff
@@ -362,3 +364,92 @@ def destroy(self, request, *args, **kwargs):
             )
         bike.unblock()
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class MalfunctionViewSet(
+    CreateModelMixin,
+    ListModelMixin,
+    DestroyModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = Malfunction.objects.all()
+    serializer_class = MalfunctionSerializer
+    request_serializer = CreateMalfunctionSerializer
+    message_serializer = MessageSerializer
+
+    @swagger_auto_schema(
+        request_body=request_serializer,
+        responses={
+            201: openapi.Response("Successful response", serializer_class),
+            400: openapi.Response("Bad request", message_serializer),
```

Comment:
Nie znam się na tym ale czy nie powinno być jeszcze tutaj wpisane 204 z DELETE'a?

Reply:
To jest endpoint `POST`, do `DELETE` masz funkcję `destroy` na samym dole